test/integration/admin/api/registry/policies_controller_test.rb

```diff
@@ -19,6 +19,15 @@ def setup
     policy_params[:policy].each { |key, value| assert_equal(value, policy.public_send(key)) }
   end
 
+  test 'POST create responds with an error message when it is incorrect' do
+    FactoryBot.create(:policy, policy_params[:policy].merge(account: @provider))
```

Comment:
This line is adding the new policy so that the POST request fails, isn't it?

Reply:
Yes basically saying you cannot create the same policy